app/src/main/java/org/p2p/wallet/receive/renbtc/ReceiveRenBtcFragment.kt

```diff
@@ -154,4 +155,10 @@ class ReceiveRenBtcFragment :
     override fun showStatuses() {
         replaceFragment(RenTransactionsFragment.create())
     }
+
+    override fun showErrorMessage(e: Throwable?) {
+        showErrorDialog(e) {
+            popBackStack()
+        }
+    }
```

Comment:
Кодстайл и сейчас, наверное, не очень важно, но я бы писал
```suggestion
    override fun showErrorMessage(e: Throwable?) = showErrorDialog(e) {
        popBackStack()
    }
```